temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingWorkflowClientCallsInterceptor.java

```diff
@@ -119,7 +120,7 @@ public <R> QueryOutput<R> query(QueryInput<R> input) {
   }
 
   @Override
-  public <R> StartUpdateOutput<R> startUpdate(StartUpdateInput<R> input) {
+  public <R> UpdateHandle<R> startUpdate(StartUpdateInput<R> input) {
```

Comment:
Hm I wonder if we would ever need to return more then `UpdateHandle` here? I think other SDKs just return `UpdateHandle` so if we do will have to address in all SDKs